src/api/user/middlewares.js

```diff
@@ -0,0 +1,33 @@
+const userService = require("../../services/user");
+const jwt = require("jsonwebtoken");
+
+function isAdmin(req, res, next) {
+  const authHeader = req.headers["authorization"];
+  if (!authHeader) {
+    return res.sendStatus(401);
+  }
+
+  const token = authHeader && authHeader.split(" ")[1];
+
+  if (!token) {
+    return res.sendStatus(401);
+  }
+
+  jwt.verify(token, process.env.ACCESS_TOKEN_SECRET, async (err, user) => {
+    if (err) {
+      return res.sendStatus(403);
+    }
```

Comment:
You shouldn't need to repeat this, the auth middleware should handle this and inject userId in the req object

Reply:
Okay, i guess i didn't understand you well when we were discussing this on skype. I removed some repeating parts, but i see now that i had to remove more.